pkg/views/member/list/view.go

```diff
@@ -0,0 +1,81 @@
+package list
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/charmbracelet/bubbles/table"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/models"
+	"github.com/goharbor/harbor-cli/pkg/utils"
+	"github.com/goharbor/harbor-cli/pkg/views/base/tablelist"
+)
+
+type model struct {
+	table table.Model
+}
+
+var columns = []table.Column{
+	{Title: "ID", Width: 4},
+	{Title: "Member Name", Width: 12},
+	{Title: "Project ID", Width: 12},
+	{Title: "Role", Width: 16},
+}
+
+var columnsWide = []table.Column{
```

Comment:
donot maintain it separately just keep it with in column itself, based on the the verbose or v flag you can change the length of col

Reply:
Since the order of the column changes I maintained it separately.

Will change the order of columns and remove it.

Reply:
Done !